### DISCUSSION TITLE
how do i get relevant logs on macOS

### DISCUSSION THREAD

**Discussion by belfie13** [helge17/tuxguitar#581] [General]
1.6.4 keeps freezing, usually after i open another music app like logic pro.

how do i get some logs we can look at to figure out what's happening?

macOS 12

**Comment by guiv42** [comment]
Might be a problem of conflict to access midi system?
Just start tuxguitar from a console, and look if errors are displayed (exceptions, ...).

**Reply by belfie13** [reply]
oh, i didnt even know you could run it from terminal
